### PR TITLE
feat: connect mfe and bff

### PR DESCRIPTION
**Pull request by leangseu-edx**
Make api call to mock BFF server.

### CODE REVIEW

File: src/containers/ResponseDisplay/__snapshots__/index.test.jsx.snap [hunk 1]
```diff
@@ -19,11 +21,6 @@ exports[`ResponseDisplay component snapshot file upload disabled without respons
   <SubmissionFiles
```
Comment (muselesscreator) [inline src/containers/ResponseDisplay/__snapshots__/index.test.jsx.snap:19]: This shouldn't be there.  Need to check that `el.instance().render()` matches snapshot, if you update props on the element.

File: src/data/redux/grading/reducer.js [hunk 2]
```diff
@@ -223,6 +222,7 @@ const grading = createSlice({
         current: {
           ...state.current,
           lockStatus: lockStatuses.unlocked,
+          gradeStatus: gradeStatuses.ungraded,
```
Comment (muselesscreator) [inline src/data/redux/grading/reducer.js]: This shouldn't be there.  Can go back to "previously graded"

File: src/data/services/lms/api.js [hunk 3]
```diff
@@ -38,11 +26,19 @@ const mockFailure = (returnValFn) => (...args) => (
...
-  courseMetadata: fakeData.courseMetadata,
-  submissions: fakeData.submissions,
-}));
+const initializeApp = () => new Promise((resolve, reject) => {
+  get(
+    stringifyUrl(urls.oraInitializeUrl, {
+      [paramKeys.oraLocation]: locationId,
+    }),
+  )
+    .then((response) => {
```
Comment (muselesscreator) [inline src/data/services/lms/api.js]: minor nit, but:
```js
  .then((response) => resolve(response.data))
  .catch((err) => reject(err));
```

File: src/data/services/lms/urls.js [hunk 4]
```diff
@@ -4,6 +4,11 @@ import { configuration } from 'config';
 const baseUrl = `${configuration.LMS_BASE_URL}`;
 
 const api = `${baseUrl}/api/`;
+const oraInitializeUrl = `${api}ora_staff_grader/mock/initialize`;
```
Comment (muselesscreator) [inline src/data/services/lms/urls.js]: I would store the mock api for less copy-pasta 
```js
const mockApi = `${api}ora_staff_grader/mock/`;
```

File: src/containers/ResponseDisplay/__snapshots__/index.test.jsx.snap (hunk 1 above)
Comment (leangseu-edx) [reply src/containers/ResponseDisplay/__snapshots__/index.test.jsx.snap:19]: It should be here, this one has response.

File: src/data/services/lms/constants.js [hunk 5]
```diff
@@ -29,3 +29,8 @@ export const fileUploadResponseOptions = StrictDict({
   optional: 'optional',
   none: 'none',
 });
+
+export const paramKeys = StrictDict({
+  oraLocation: 'oraLocation',
+  submissionUUID: 'submissionUUID',
+});
```
Comment (nsprenkle) [inline src/data/services/lms/constants.js:36]: Love storing them here as constants

File: src/containers/ResponseDisplay/__snapshots__/index.test.jsx.snap (hunk 1 above)
Comment (muselesscreator) [reply src/containers/ResponseDisplay/__snapshots__/index.test.jsx.snap:19]: says without response in title, and the test doesn't have one?

**Review by muselesscreator — APPROVED** [review]
LGTM

**Review by jnlapierre — APPROVED** [review]
LGTM

**Pull request merged**